File: maximum-product-difference-between-two-pairs/maximum-product-difference-between-two-pairs.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var maxProductDifference = function(nums) {
  const sorted = nums.sort((a, b) => b - a);
  return (sorted[0] * sorted[1]) - (sorted[sorted.length - 2] * sorted[sorted.length - 1]);
};

File: maximum-product-difference-between-two-pairs/maximum-product-difference-between-two-pairs.py
class Solution:
    def maxProductDifference(self, nums: List[int]) -> int:
      sortedNums = sorted(nums)
      return (sortedNums[len(nums) - 1] * sortedNums[len(nums) - 2]) - (sortedNums[0] * sortedNums[1])